hpx/plugins/plugin_registry.hpp

```diff
@@ -89,15 +87,32 @@ namespace hpx { namespace plugins
     )(__VA_ARGS__))                                                           \
 /**/
 
-#define HPX_REGISTER_PLUGIN_REGISTRY_2(PluginType, pluginname)                \
-    typedef hpx::plugins::plugin_registry<PluginType>                         \
-        pluginname ## _plugin_registry_type;                                  \
-    HPX_REGISTER_PLUGIN_BASE_REGISTRY(                                        \
-        pluginname ## _plugin_registry_type, pluginname)                      \
-    HPX_DEF_UNIQUE_PLUGIN_NAME(                                               \
-        pluginname ## _plugin_registry_type, pluginname)                      \
-    template struct hpx::plugins::plugin_registry<PluginType >;               \
-/**/
+#define HPX_REGISTER_PLUGIN_REGISTRY_2(PluginType, pluginname)                 \
+    HPX_REGISTER_PLUGIN_REGISTRY_5(                                            \
+        PluginType, pluginname, HPX_PLUGIN_NAME, "hpx", "hpx")                 \
+    /**/
+#define HPX_REGISTER_PLUGIN_REGISTRY_4(                                        \
+    PluginType, pluginname, pluginsection, pluginsuffix)                       \
+    HPX_REGISTER_PLUGIN_REGISTRY_5(PluginType, pluginname, HPX_PLUGIN_NAME,    \
+        pluginsection, "pluginsuffix")                                         \
```

Comment:
why is `pluginsuffix` quoted here?

Reply:
Yep, caught that yesterday as well.... Should be fine now.